AGENTS.md

```
@@ -159,3 +159,35 @@ When finishing a development branch:
+| Localization/translation updates | 1 |
+| Dependency version bump | 1 |
+| Internal component refactor (no UI change) | 2 |
+| New UI component (non-critical path) | 2 |
+| User-facing UI flow changes (chat, settings) | 3 |
+| Console plugin API integration changes | 3 |
+
+### Jira Description Format
+
+Include this section in every ticket description:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)